From 8dfb84ad4efbc39c7a7d9efdff6f6ac9017e0a53 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Thu, 6 Nov 2025 14:33:22 +0100
Subject: [PATCH 46/48] support: Fix FILE * leak in check_for_unshare_hints in
 test-container

The file opened via fopen is never closed.

(cherry picked from commit 20a2a756089eacd7e7f4c02e381e82b5d0e40a2c)
---
 support/test-container.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/support/test-container.c b/support/test-container.c
index 1c40ab377f..d78139622f 100644
--- a/support/test-container.c
+++ b/support/test-container.c
@@ -705,6 +705,7 @@ check_for_unshare_hints (int require_pidns)
 
       val = -1; /* Sentinel.  */
       int cnt = fscanf (f, "%d", &val);
+      fclose (f);
       if (cnt == 1 && val != files[i].bad_value)
 	continue;
 
-- 
2.51.2

